common/scripted_triggers/wc_cultural_traditions_trigger.txt

```diff
@@ -20,4 +20,39 @@ can_have_tradition_runestones_trigger = {
 		has_cultural_pillar = heritage_vrykulic
 		has_cultural_pillar = heritage_jalgaric
 	}
+}
+can_have_tradition_seafaring_trigger = {
+	OR = {
+		is_culture_or_parent_culture_trigger = { CULTURE = culture:baradin }
+		is_culture_or_parent_culture_trigger = { CULTURE = culture:tirassian }
+		is_culture_or_parent_culture_trigger = { CULTURE = culture:wandering_pandaren }
+		is_culture_or_parent_culture_trigger = { CULTURE = culture:pirate_culture }
+		has_cultural_pillar = heritage_vrykulic
+		has_cultural_pillar = heritage_goblin
+		has_cultural_pillar = heritage_chelonian
+		
+		# Default WC children cultures will not inherit
+		this = culture:stormreaver
+		this = culture:zandalari
```

Comment:
Why? I think it's fine. It will be weird for the players if the tradition is gone.

Reply:
In the case of the Stormreavers, their sea-going ways were directed specifically by Gul'dan for the Second War and later, the voyage to the Broken Isles. It's not clear if this cultural tradition would have been maintained after this mission, especially since the Stormreavers were an artificial clan of opportunistic megalomaniacs drawn together by Gul'dan's promises of power and not anything else they had in common. In the main universe canon, the clan promptly disintegrated once it lost Gul'dan's leadership.

The reason Zandalari is in this category is cause none of their children cultures (Amani, Farraki, Drakkari, etc) have retained sea-going practices. That said, those cultures are further removed from Zandalari than a newly diverged one during a GoA game, so it also makes sense to allow children cultures to keep it.

Reply:
Resolved after trigger deletion.